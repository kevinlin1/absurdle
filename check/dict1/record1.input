crate
loins
gully
bully
dully
fully
moody
howdy
