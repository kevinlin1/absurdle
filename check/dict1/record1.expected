2309 words
> crate
: ⬜⬜⬜⬜⬜

246 words
> loins
: ⬜🟩⬜⬜⬜

16 words
> gully
: ⬜⬜⬜⬜🟩

10 words
> bully
: ⬜⬜⬜⬜🟩

6 words
> dully
: 🟨⬜⬜⬜🟩

3 words
> fully
: ⬜⬜⬜⬜🟩

3 words
> moody
: ⬜🟩⬜🟩🟩

1 words
> howdy
: 🟩🟩🟩🟩🟩

Absurdle 8/∞

⬜⬜⬜⬜⬜
⬜🟩⬜⬜⬜
⬜⬜⬜⬜🟩
⬜⬜⬜⬜🟩
🟨⬜⬜⬜🟩
⬜⬜⬜⬜🟩
⬜🟩⬜🟩🟩
🟩🟩🟩🟩🟩
